content/frontend/react-apollo/7-filtering-searching-the-list-of-links.md

```diff
@@ -179,7 +179,7 @@ In this case, you're specifying two filters that account for the following two c
 
 Perfect, the query is defined! But this time we actually want to load the data every time the user hits the _search_-button. 
 
-That's what you're using the [`withApollo`](http://dev.apollodata.com/react/higher-order-components.html#withApollo) function for. All it does is injecting a new prop into the `Search` component called `client`. This `client` is precisely the `ApolloClient` instance that you're creating in `index.js` and that's now directly available inside `Search`.
+That's what you're using the [`withApollo`](http://dev.apollodata.com/react/higher-order-components.html#withApollo) function for. This function injects a new prop into the `Search` component called `client`. This `client` is precisely the `ApolloClient` instance that you're creating in `index.js` and which is now directly available inside `Search`.
```

Comment:
Did a deeper rewrite. Really a lot of the text in these tutorials could use a pass from an experienced copy editor, I'm just trying to fix a few of the rougher spots as I hit them.